starter-code/french-onion-soup-starter-code/css/main.css

```diff
@@ -7,3 +7,65 @@ html {
   font-family: 'Open Sans', sans-serif;
   font-size: 18px
 }
+
+h1 {
+  margin: 0 auto;
+  color: white;
+  padding-top: 400px;
+  font-size: 4em;
+  width: 100%;
+  text-align:center;
+  font-weight: 100;
+
+}
+
+
+.frenchHeader {
+  background-image: url(../images/soup.jpg);
```

Comment:
this image seemed to be a little large. You can adjust it by modifying the height of the div and the width if you like, though in this case you didn't need to adjust the width.